packages/ui/src/components/Switch/styles.ts

```diff
@@ -0,0 +1,66 @@
+import { Color } from "../../types";
+
+export type SwitchColor = Omit<
+  Color,
+  "black" | "white" | "blackFixed" | "whiteFixed"
+>;
+
+const getStyles = (
+  color: SwitchColor,
+  disabled?: boolean,
+): Record<string, string> => {
+  let colorStyle: Record<string, string> = {};
+
+  if (disabled) {
```

Comment:
This doesn't seem to be working for me - they're still all coloured.  This time Chromatic agrees with me.  :laughing: 
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/4ecd715e-ad92-488b-87c0-ba1341523928)

Reply:
Oops, I guess actually passing the prop would help 😅 

Fixed in [pass down disabled prop](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8605/commits/2e2bc134f0808352a7170e66ef8c9fbdb540d2eb)

Reply:
Oh yeah, that helps.  :laughing:   Looks great!